.github/workflows/process-created-issue.yml

```
@@ -111,7 +111,7 @@ jobs:
                       break;
                     }
                   case 'jsfiddle.net':
-                    if (/^\/.+\/.+$/.test(reproURL.pathname)) {
+                    if (/^\/.+\//.test(reproURL.pathname)) {
```

A trailing slash isn't guaranteed, and it's not important either.
```suggestion
                    if (/^\/.+/.test(reproURL.pathname)) {
```